plugins/nat/nat_windows.go

```diff
@@ -7,10 +7,11 @@ import (
 
 /*
 // NetPlugin represents the CNI network plugin.
-type netPlugin struct {
-	*cni.Plugin
-	nm network.Manager
-}
+
+	type netPlugin struct {
```

Comment:
This is shifted right which is not correct.

Reply:
That's actually `go fmt` identifying code snippets within the comment and attempting to indent it so it can get formatted as code by `godoc`.

Either way, I've noticed that these comments, initially added in [this PR](https://github.com/microsoft/windows-container-networking/commit/e9885bfd7736f2951a6088aaa2cdd6bef85d7872#diff-20c7e952829a52904d8af240eec15cc85f4c920c63168b3a624a414e0f4f3502R9), are most likely redundant since the `netPlugin` struct has [been moved into its own package](https://github.com/microsoft/windows-container-networking/blob/master/common/core/network.go#L25-L29), so I'll be removing them completely.